middlewares/authenticator.go

```diff
@@ -88,6 +111,14 @@ func parserDigestUsers(users types.Users) (map[string]string, error) {
 	return userMap, nil
 }
 
+func getUserStringsFromFile(filename string) ([]string, error) {
```

Comment:
I'm not aware that we have any (yet). Did a real quick look but couldn't find anything.

I'd say let's keep it in this package for now. Once we see similar call needs surfacing, we can extract into a new library package.